cli/dcoscli/data/help/package.txt

```diff
@@ -68,6 +71,9 @@ Options:
         Print a short description of this subcommand.
     --options=<file>
         Path to a JSON file that contains customized package installation options.
+    --output-directory=<output-directory>
+        Path to the directory where the data should be stored.
+        Defaults to target/ underneath the directory of the DC/OS Package Build Definition.
```

Comment:
@larioj What do you think of "... the directory of the <build-definition> file" instead of "... the directory of the DC/OS ..."?